lib/ascii-art.js

```diff
@@ -0,0 +1,33 @@
+const {CompositeDisposable} = require('atom');
```

Comment:
For consistency with [Atom's JavaScript style guide](https://github.com/atom/atom/blob/34cd21db04dd522e2b08b19c5af0123620bcc09a/CONTRIBUTING.md#javascript-styleguide), you'll want to convert this to JavaScript Standard Style. To do so, you can to root of the repository and run:

```
$ npm install -g standard
$ standard --fix
```

Then, fix any linter errors that `standard` reports, and commit the changes.